backend/utility/emailTemplates.js

```diff
@@ -0,0 +1,14 @@
+// This function will be used to generate a template which will be utilized in all email types
+// module.exports.generateHtml(data){};
+
+module.exports.getMailTemplate = (type) => {
+  const templates = {
+    'INVITE-ADMIN': {
+      // html:"",
+      subject: '[INVITATION] Invited to become an admin',
+      text:
+        'Hi There!\n We would love for you to join us at HITK Tech Community as an admin.\n\tHITK Tech Community is a platform built by the students and for the students with the main intent of increasing awareness towards plethora of opportunities and internships in tech all around/over the year. This will not only give practical work experience/exposure to students, but will also help everyone to know and grab their required opportunities in time!',
```

Comment:
hey, this will also contain variables like the ones given in the sample raw template. Can you please incorporate a mechanism to use it

Refer to https://github.com/HITK-TECH-Community/Community-Website/pull/256 for more context